packages/fuel-indexer/src/service.rs

```diff
@@ -286,3 +299,23 @@ async fn create_service_task(
         }
     }
 }
+
+async fn get_start_block(
+    conn: &mut IndexerConnection,
+    manifest: &Manifest,
+) -> Option<u64> {
+    let resumable = manifest.resumable.unwrap_or(false);
+    if resumable {
+        let last_block = queries::last_block_height_for_indexer(
+            conn,
+            &manifest.namespace,
+            &manifest.identifier,
+        )
+        .await
+        .unwrap_or(1) as u64;
+        println!("Last block: {}", last_block);
```

Comment:
- If there's info we want the user aware of we can use `logger::info`
  - But in this particular case I think we can remove this `println!` as it isn't too informative